userspace/libpman/src/configuration.c

```diff
@@ -88,6 +104,11 @@ int pman_init_state(falcosecurity_log_fn log_fn, unsigned long buf_bytes_dim, ui
 
 	/* Set libbpf logging. */
 	g_state.log_fn = log_fn;
+	// we allocate a big buffer for verifier logs we will free it after initialization.
+	g_state.log_buf = calloc(1, BPF_LOG_BIG_BUF_SIZE);
```

Comment:
should we reset `log_buf_size` here like on line 51 or is it guaranteed to be already 0?

Reply:
we are setting it at line `111` to `BPF_LOG_BIG_BUF_SIZE` if the `calloc` call was successful 
```c
	g_state.log_buf = calloc(1, BPF_LOG_BIG_BUF_SIZE);
	if(g_state.log_buf == NULL)
		return -ENOMEM;
	g_state.log_buf_size = BPF_LOG_BIG_BUF_SIZE;
	libbpf_set_print(libbpf_print);
```

Reply:
at line 51 we continue the execution in case of failure that's why we reset it to `0`

Reply:
thanks!